### PR TITLE
filter by acronym

### PR DESCRIPTION
**Pull request by nlkluth**

### CODE REVIEW

**Comment by NFabrizio** [comment]
I don't know why it's not showing in the diff, but this branch does not have the material-ui upgrade that's in master, so running `npm run build` fails on this branch.

File: src/Home.js [hunk 1]
```diff
@@ -64,14 +77,49 @@ class Home extends Component {
       return 'loading...';
     }
 
+    const definitionMatch = R.compose(
+      R.any(R.contains(this.state.search)),
```
Comment (NFabrizio) [inline src/Home.js]: Do we need to use `any` here? Won't `chain` return a single array?

Comment (NFabrizio) [reply src/Home.js]: Nevermind. I see why now, but should we add a comment so we remember why we need this later?

Comment (nlkluth) [reply src/Home.js]: You have a comment in mind? 😄

**Pull request merged**